### ISSUE TITLE
Add features to #5 and #16 (stochastic and bit parity tests)

### ISSUE THREAD

**Issue by alessandramaranca** [niznik-dev/predicting-zygosity#38]
Add the following features to the probabilistic (#5) and stochastic (#16) tests:

1. Make it clear in the plot whether we are evaluating on test or training set
2. Make it such that n (sequence length) and N (sample size) are different by introducing sampling with replacement.
3. Plot distributions of P(0) (probability associated with 0) and P(1) (probability associated with 1), and check if P(0) and P(1) sum up to 1. 
4. Think about the saving epochs issues
5. Add to the workflow test of whether it is fine-tuning (#37) before evaluation

More context: https://docs.google.com/document/d/1rZFhzmBDRbMvJRqs1Z0sv4R-7iFoC-fqDrY0NT4JRiA/edit?usp=sharing

**Comment by alessandramaranca** [comment]
Suggestions from Jun 3rd meeting:

- Make a movie epoch by epoch
- Sample from input and output vs sampling from input (for generalizing N)
- Flip 0 and 1 
- Average probability distribution over p
- Look into normalization (are we evaluating differently here)?
